Makefile

```diff
@@ -17,7 +17,8 @@ install-test-tools: all
 	go install github.com/google/seesaw/test_tools/quagga_test_tool
 
 proto:
-	protoc --go_out=. pb/config/config.proto
+	go install google.golang.org/protobuf/cmd/protoc-gen-go
+	cd pb/config; protoc --go_out=paths=source_relative:. config.proto
```

Comment:
Add a similar line for pb/seesaw.

Reply:
@DrJosh9000 Thanks. Missed on that one. Just pushed another commit on top.